### DISCUSSION TITLE
How to Remove Conditions?

### DISCUSSION THREAD

**Discussion by Quest79** [BAndysc/WoWDatabaseEditor#70] [Q&A] [answered]
Aside from making a manual SQL query, is there a way to delete conditions? Ive right-clicked on conditions and left-clicked and looked at all the options, there is not "remove" feature. Is this missing or am I missing it?

- Id like to add to the below answer in that you can **ctrl+c ctrl+v** stuff too. This is very good UX.

**Comment by BAndysc** [comment, marked as answer]
I should probably add it to the edit menu, but "delete" or "backspace" keys work already now

**Reply by Quest79** [reply]
They do! I am blind, thanks
